rules/react.js

```diff
@@ -0,0 +1,68 @@
+// eslint-plugin-react https://github.com/yannickcr/eslint-plugin-react
+
+module.exports = {
+	"plugins": [ "react" ],
+	"rules": {
+		"react/display-name": [ 2, { "acceptTranspilerName": true } ], // Prevent missing displayName in a React component definition
+		"react/forbid-prop-types": [ 2, { "forbid": [ "any" ] } ], // Forbid certain propTypes
+		"react/jsx-boolean-value": [ 1, "always" ], // Enforce boolean attributes notation in JSX
+		"react/jsx-closing-bracket-location": [ 1, { "location": "after-props" } ], // Validate closing bracket location in JSX
+		"react/jsx-curly-spacing": [ 2, "always" ], // Enforce or disallow spaces inside of curly braces in JSX attributes
+		"react/jsx-handler-names": [ 1, { "eventHandlerPrefix": "handle", "eventHandlerPropPrefix": "on" } ], // Enforce event handler naming conventions in JSX
```

Comment:
Discuss, do we want this to be ignored (0) or to be an error (2)?

https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-handler-names.md